### PR TITLE
two additional remarks in documentation

### PR DESCRIPTION
**Pull request by Konfekt**
These two points did not seem immediately clear to everyone

### CODE REVIEW

File: runtime/doc/builtin.txt [hunk 1]
```diff
@@ -8991,11 +8991,12 @@ shellescape({string} [, {special}])			*shellescape()*
...
 
-		When the {special} argument is present and it's a non-zero
-		Number or a non-empty String (|non-zero-arg|), then special
-		items such as "!", "%", "#" and "<cword>" will be preceded by
-		a backslash.  This backslash will be removed again by the |:!|
-		command.
+		The {special} argument adds additional escaping of keywords
+		used in Vim commands. When it is not omitted and a non-zero
+		number or a non-empty String (|non-zero-arg|), then special 
+		words such as "!", "%", "#" and "<cword>" (as listed in
```
Comment (chrisbra) [inline runtime/doc/builtin.txt]: can you keep the `items` instead of `words` please?

File: runtime/doc/builtin.txt [hunk 2]
```diff
@@ -10242,7 +10243,8 @@ tempname()					*tempname()* *temp-file-name*
 		is different for at least 26 consecutive calls.  Example: >
 			:let tmpfile = tempname()
 			:exe "redir > " .. tmpfile
-<		For Unix, the file will be in a private directory |tempfile|.
+<		For Unix, the file will be in a private directory |tempfile|
```
Comment (Konfekt) [inline runtime/doc/builtin.txt:10246]: Somehow this comment was dropped in https://github.com/vim/vim/commit/5faeb60480c6efba5c0468c01275120b6ace5a09
Is this clear to everyone? I got caught off-guard once and kept wondering for a while.

Comment (chrisbra) [reply runtime/doc/builtin.txt:10246]: It didn't get dropped, I just added the missing trailing dot back and then there was no change anymore :)

Comment (Konfekt) [reply runtime/doc/builtin.txt:10246]: When looking at https://github.com/vim/vim/pull/14770/files there's also https://github.com/vim/vim/pull/14770/files#diff-ef8946d89919ad628b9ac65c49d3067e4e017b0980cac70eb25d832a59ef677dR10247 that was meant to warn against recursive deletion of everything inside the temp dir...

Comment (chrisbra) [reply runtime/doc/builtin.txt:10246]: hm, I might have missed to squash the third commit, sorry, let me fix it

Comment (chrisbra) [reply runtime/doc/builtin.txt:10246]: updated and further clarified here: 5cf5301e2847c5a1a380ae030c58825b5c6fe275

**Pull request closed without merging**